src/arithmetic.jl

```diff
@@ -1,27 +1,19 @@
-import Base: +, -
-import Compat: @static
-
-if VERSION < v"0.7.0-DEV.4955"
-    import Base: broadcast
-    const broadcasted = broadcast
-else
-    import Base.Broadcast: broadcasted
-end
+import Base.Broadcast: broadcasted
```

Comment:
If this is being extended can you just use the module annotation there rather than `import`ing it?

Reply:
In this particular case I thought the `import` made things a bit clearer. I'm not a fan of `Base.Broadcast.broadcasted(...) = `

Reply:
That's fair. Though you could also do
```julia
using Base.Broadcast

Broadcast.broadcasted(...) =
```

Reply:
That's a good point. I do like that but I don't want o wait for the CI again 😆